code/modules/client/preferences.dm

```diff
@@ -118,6 +118,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				if(initial(lang_path.secret))
 					languages.Remove(lang_path)
 			// SKYRAT EDIT END
+			return // SKYRAT EDIT - Don't remove this. Just don't. Nothing is worth forced random characters.
```

Comment:
this isn't really a fix, i think we should figure out why character data isn't loading instead

Reply:
it is a fix, because if it DID load it without this it would just randomise it anyway. The bug was that it randomised it no matter what, not that character data didnt load

Reply:
ah okay